example_project/measures/BuildResidentialModel/tests/test_build_residential_model.rb

```diff
@@ -293,16 +293,29 @@ def test_residential_templates
 
   def test_residential_samples
     feature_buildstock_csv_paths = [File.absolute_path(File.join(File.dirname(__FILE__), '../../../resources/residential-measures/test/base_results/baseline/annual/buildstock.csv'))]
-    resstock_building_ids = [1, 3, 8, 15] # SFD, Mobile Home, SFA, MF
+    feature_building_types = ['Single-Family Detached', 'Single-Family Attached', 'Multifamily']
+    feature_number_of_residential_unitss = [1, 5, 7]
+    feature_floor_areas = [5000, 9000]
```

Comment:
Actually test returning a `resstock_building_id` based on combinations of feature properties.